### PR TITLE
[risk=no] Survey synonyms

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Adding a column to have survey synonyms. 
Having this column makes it easier to search for search word in either the question text / any of the answers.

### CODE REVIEW

File: public-api/db-cdr/changelog-schema/db.changelog-68.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <indexExists tableName="question_concept" indexName="question_string_index"></indexExists>
+      </not>
+    </preConditions>
+    <sql dbms="mysql">
+      ALTER TABLE question_concept
+      ADD FULLTEXT INDEX question_string_index
+      (question_string);
+    </sql>
```
Comment (freemabd) [inline public-api/db-cdr/changelog-schema/db.changelog-68.xml]: you could just add the index to the changset above -> db.changelog-67 and delete this file all together

File: public-api/db-cdr/changelog-schema/db.changelog-master.xml [hunk 2]
```diff
@@ -64,4 +64,6 @@
   <include file="changelog-schema/db.changelog-64.xml"/>
   <include file="changelog-schema/db.changelog-65.xml"/>
   <include file="changelog-schema/db.changelog-66.xml"/>
+  <include file="changelog-schema/db.changelog-67.xml"/>
+  <include file="changelog-schema/db.changelog-68.xml"/>
```
Comment (freemabd) [inline public-api/db-cdr/changelog-schema/db.changelog-master.xml]: could be 1 file instead of 2

**Review by freemabd — APPROVED** [review]
A few comments about consolidating your liquibase changesets, but looks good.

**Review by freemabd — APPROVED** [review]
Left a few comments about consolidating liquibase changsets, but looks good.

**Comment by SrushtiGangireddy** [comment]
Thanks.

**Pull request merged**